src/Sentry/SentryOptions.cs

```diff
@@ -746,6 +746,12 @@ public IList<SubstringOrRegexPattern> FailedRequestTargets
         true => TracesSampler is not null || TracesSampleRate is > 0.0 or null
     };
 
+    /// <summary>
+    /// Indicates whether profiling is enabled, via any combination of
+    /// <see cref="EnableTracing"/>, <see cref="TracesSampleRate"/>, or <see cref="TracesSampler"/>.
+    /// </summary>
+    internal bool IsProfilingEnabled => IsPerformanceMonitoringEnabled && ProfilesSampleRate > 0.0;
```

Comment:
Might be a weird hill to die on but could we go with `EnableProfiling` for consistency?

Reply:
That'd be even better.